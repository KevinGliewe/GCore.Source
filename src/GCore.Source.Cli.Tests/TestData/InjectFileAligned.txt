<[Script Dst="..:out"]>
Writer.WriteLine("|Overwrite|");
Writer.Write("->||<-");
<[/Script]>
<[RawAligned Name="out" AlignTag="|"]>
This gets overwritten
<[/RawAligned]>