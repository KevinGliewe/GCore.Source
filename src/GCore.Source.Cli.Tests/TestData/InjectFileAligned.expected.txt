<[Script Dst="..:out"]>
Writer.WriteLine("|Overwrite|");
Writer.Write("->||<-");
<[/Script]>
<[RawAligned Name="out" AlignTag="|"]>
    Overwrite
->              <-
<[/RawAligned]>